package.json

```diff
@@ -18,8 +18,8 @@
         "findup-sync": "^5.0.0",
         "fs-extra": "^11.1.1",
         "logplease": "^1.2.15",
-        "semver": "^7.5.1",
-        "solc-typed-ast": "^15.0.0",
+        "semver": "^7.5.4",
+        "solc-typed-ast": "file:../solc-typed-ast",
```

Comment:
Just a reminder to fix before final approval

Reply:
Fixed in 412160a (v16 is released now).